_schemaV2.graphql

```diff
@@ -7263,6 +7268,23 @@ type Me implements Node {
   unreadNotificationsCount: Int!
 }
 
+# Collection of fields that describe medium type, such as _Painting_. (This field
+# is also commonly referred to as just "medium", but should not be confused with
+# the artwork attribute called `medium`.)
+type MediumType {
```

Comment:
#veryminor could maybe just be `type Medium` (or maybe even `type ArtworkMedium`, since just 'medium' can be maybe ambiguous (is it a size?)), and also the `...Type` suffix is a bit redundant

Reply:
We do have a `medium` attribute on Artwork today which refers to the material used. `Artwork#artworkMedium` sounds a bit redundant to me. Happy to rename the type but are you also suggesting renaming the Artwork type field?

Reply:
This is the name of the type, so I don't think should be conflated with the naming of a field which is of this type (or even related fields like `medium`). The current `medium` _field_ in the artwork schema is a string type.

Basically I think you can adjust this type name (if warranted), w/o worrying about any field names (that's a separate thign).